UPGRADE-5.0.md

```diff
@@ -393,7 +393,7 @@ Routing
 -------
 
  * The `generator_base_class`, `generator_cache_class`, `matcher_base_class`, and `matcher_cache_class` router
-   options have been removed.
+   options have been removed. If you are using multiple Router instances and need separate caches for them, set a unique `cache_dir` per Router instance instead
```

Comment:
Missing `.` + this should target `4.4` :)

Reply:
I can add the `.`, but how do I change the target?

Reply:
You have to rebase your branch on 4.4 + push + update the target branch of the PR to 4.4.

Reply:
ok, that sounds incredibly tedious, so I'll probably just close this and open a new PR (from what Google tells me, the recipient can somehow change the base branch in the github UI, but I'm honestly too lazy to figure out how to do it on the sender side). Should I still modify `UPGRADE-5.0.md`, or should the change rather be in `UPGRADE-4.4.md` ?

Reply:
Logically you just have to do `git rebase --onto 4.4 master` + `git push origin patch-1 --force` + edit the PR to change the target.

> Should I still modify UPGRADE-5.0.md, or should the change rather be in UPGRADE-4.4.md ?

UPGRADE-5.0.md

Reply:
`git rebase --onto 4.4 master` was what I found on Google, too, but if I run it locally (with branch `patch-1` checked out), I only get

```
fatal: '4.4' does not point to a valid Commit.
```
(error message translated from German). Maybe my local `git` is too old or something (I have 2.20.1)?

Reply:
if you haven't created a local 4.4 branch yet, you can change the command to reference `origin/4.4` instead

Reply:
@stof Thanks, this seems to have done the trick! Unfortunately, I must have broken something else, since now I also have 6 commits by other people in my PR. Sorry, my git-foo really sucks. I still miss SVN... :-)

Reply:
that's because you haven't done the third step of the process yet: editing your PR on Github to change the base branch (and so commits which are in the 4.4 branch but not yet in master appear in the diff between your branch and master, which is expected)

Reply:
ah, ok, so can can change the base from the sender side. I didn't get that from github documentation (which I admittedly only skimmed..). Should be fixed now. The missing `.` is also added (and squashed into a single commit, since I figured you probably wouldn't want two commits to add a single sentence)